docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,575 @@
+
+### Benefits
+
+Global profile configuration can improve your Zowe CLI experience in the following ways:
+
+- As a CLI user, you can manage your connection details efficiently in one location.
+- As a team leader, you can share a configuration file with your team members so that they can easily access mainframe services. You can add the file directly to your project in an SCM.
+- As a new team member, you can onboard quickly by consuming your team's configuration file.
+
+### Changes to secure credential storage
+
+In this version, Secure Credential Store (SCS) Plug-in is deprecated. The `zowe scs` and `zowe config` command groups are obsolete. The equivalent functionality that encrypts your credentials is now included in the core CLI.
+
+With the new configuration, the CLI prompts you to enter username and password securely by default. Commands in the new `zowe config` command group let you manage security for any option value.
+
+## Installing @next version
+
+To get started, install the Zowe CLI `@next` version from the online registry. You can follow this procedure for a first-time installation, or to update a currently installed version.
+
+**Follow these steps:**
+
+1. Meet the [software requirements for Zowe CLI](https://docs.zowe.org/stable/user-guide/systemrequirements.html#zowe-cli-requirements).
+
+2. To install or update the core CLI, open a command-line window and issue the following command:
+
+   ```
+   npm install -g @zowe/cli@next
+   ```
+
+3. Meet the [software requirements for each plug-in](https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html#software-requirements-for-zowe-cli-plug-ins).
+
+4. (Optional) Check [npmjs.com](https://www.npmjs.com/) for any Zowe plug-ins that have an `@next` version available. If an `@next` version is available, you can issue the following command (substituting the plugin name) to install it. If no `@next` version, try installing the `@latest` version of the plug-in:
+
+    ```
+    zowe plugins install @zowe/<plugin-name>@next
+    ```
+
+   Zowe CLI and optional plug-ins are installed!
+
+5. If you previously had an instance of Zowe CLI installed, your old configuration files are no longer used in this version. Delete the following files from your local `.zowe/` directory:
+   - `.zowe/settings/imperative.json`
+   - `.zowe/profiles`
+
+   **Important!** Prior to deleting the contents of the `/profiles` directory, take note of any mainframe service details that you need (host, port, etc...). You might want to save the entire `/profiles` directory to another location on your computer so that you can reference or restore the profiles later.
+
+6. If you previously had the Secure Credential Store plug-in installed, uninstall it now to avoid unexpected behavior. Issue the following command:
```

Comment:
Should we swap #5 and #6, and suggest that before uninstalling SCS, users run `zowe scs revert --force`? This would revert all credentials to plain text before profiles are deleted. If users skip this and delete their profiles first, then their vault will be cluttered by unused credential entries.